neptune/client.py

```diff
@@ -110,6 +111,15 @@ def __init__(self, api_address, api_token):
         )
         self._http_client.authenticator = self.authenticator
 
+    @with_api_exceptions_handler
```

Comment:
I don't think we need this decorator here, since the method doesn't make any calls.